uring/jvm/src/main/scala/fs2/io/uring/unsafe/UringSystem.scala

```diff
@@ -326,6 +344,8 @@ object UringSystem extends PollingSystem {
 
       pendingSubmissions = false
 
+      end()
```

Comment:
We should put the `end` in a `finally` block.

https://docs.oracle.com/javase%2F7%2Fdocs%2Fapi%2F%2F/java/nio/channels/spi/AbstractSelector.html